torch/testing/_internal/opinfo/core.py

```diff
@@ -1228,6 +1238,9 @@ def non_empty_sampler(op, generator):
             sample_inputs_mth(device, dtype, requires_grad=requires_grad, **kwargs),
         )
 
+    def _sample_inputs_unspecified(self, *args, **kwargs):
+        raise unittest.SkipTest("NO SAMPLE FUNCTION SPECIFIED!")
```

Comment:
So we'll see a lot of skips from now on?

Reply:
No, the number of such skips is relatively small (<100 out of tens of thousands sparse specific test tasks) and all such skips indicate that the corresponding sparse sample functions need to be implemented.

In general, there is a little point of defining that an op supports a sparse layout without providing sparse samples (although, there exist a few methods that generate the sparse samples within the methods based on the sparse layout support flag but these methods don't call the `OpInfo.sample_inputs_sparse` method).

The method `_sample_inputs_unspecified` is called only when the following conditions are satisfied all at the same time:
1. An OpInfo instance has specified `supports_sparse_<layout>=True`. Currently, there about 30-40 such OpInfo instances (the exact number depends on the layout).
2. The OpInfo instance has not specified `sample_inputs_sparse_<layout>_func`. Currently, there are 3 OpInfo instances that specify `sample_inputs_sparse_<layout>_func` out of all instances from 1.
3. A `OpInfo.sample_inputs_sparse` method is called. Currently, there are only 4 test methods in `test/test_sparse.py` that call the method.

Atm, the number of OpInfo instances for which all the three conditions hold, is 1 (this is OpInfo for `torch.randn_like`) which leads to `NO SAMPLE FUNCTION SPECIFIED` message when running the `test_like_fns` method from `test_sparse.py`.